cmd/tools/grafana/dashboard_test.go

```diff
@@ -487,6 +487,24 @@ func TestIDIsBlank(t *testing.T) {
 		})
 }
 
+func TestExemplarTrue(t *testing.T) {
```

Comment:
minor: but maybe `TestExemplarIsFalse` is better since it states the desired state instead of the problem state